src/script/E2EIdentity/E2EIdentity.ts

```diff
@@ -67,16 +68,6 @@ class E2EIHandler {
     });
   }
 
-  private get coreE2EIService() {
-    const e2eiService = this.core.service?.e2eIdentity;
-
-    if (!e2eiService) {
-      throw new Error('E2EI Service not available');
-    }
-
-    return e2eiService;
-  }
```

Comment:
why did you change this behavior?

Reply:
You can just mock the core.service.e2eIdentity globally and it wont break any tests